.gitignore

```diff
@@ -47,7 +47,6 @@ npm-debug.log
 /tsd_typings/
 
 # Dist #
-/lint
```

Comment:
Looks like this hunk needs to be deleted when rebasing onto current master, or else we'll stop ignoring this generated directory again.

Reply:
I think this was added in PR